examples/user_guide/Outputs.ipynb

```diff
@@ -0,0 +1,164 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "94463ebb",
```

Comment:
Oh no, looks like you upgraded 😄

Reply:
Happy to have the 'id' field added to what we strip out...